tests/runcheck

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+## Runs devtools::check() and checks for ERROR or WARNING in the output log.
+## Exits with code 0 if there are none, with 1 otherwise.
+
```

Comment:
This might could profit from travis fold statements, though I can understand if we don't want them leaking everywhere.